nlohmann_json.podspec

```diff
@@ -0,0 +1,89 @@
+#
+#  Be sure to run `pod spec lint nlohmann_json.podspec' to ensure this is a
+#  valid spec and to remove all comments including this before submitting the spec.
+#
+#  To learn more about Podspec attributes see http://docs.cocoapods.org/specification.html
+#  To see working Podspecs in the CocoaPods repo see https://github.com/CocoaPods/Specs/
+#
+
+Pod::Spec.new do |s|
+
+  # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  s.name         = "nlohmann_json"
+  s.version      = "3.1.2"
+  s.summary      = "JSON reader & writer for C++ optimized for ease of use and modern C++14"
```

Comment:
That's not really the summary of the library. Please use "JSON for Modern C++" or something. And the library also supports C++11.